content/ADR-175-remove-assets-from-build-local-addressables.md

```diff
@@ -0,0 +1,105 @@
+
+### Addressables loading
+Depending on the specific case, we will use all possible ways to load assets:
+  - [Label](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/Labels.html) (to load a list of similar assets)
+  - Address (to load an asset by the hardcoded string)
+  - GUID (to load an asset by "Asset Reference")
+
+### [Groups configuration](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/GroupSettings.html)
+Group settings determine how the assets in a group are treated in content builds.
+
+![image](https://user-images.githubusercontent.com/118179774/214874324-6bb9b942-dd00-4a65-bee4-213e1f4c4125.png)
+
+We are going to distribute assets in "Addressable Asset Groups" according to the following rules:
+   - "Essentials" corresponds to the assets that are mandatory for the application to function (e.g. Skybox) and can't be excluded
+   - Other assets according to the logical designation (e.g. "Fonts") and usage (e.g. "Window XXX Resources") fall into different groups; without them the application can function
+   - Assets packed together/separately according to the required loading way: any combination of "Include Addresses in Catalog", "Include GUIDs in Catalog" and "Include Labels in Catalog". Thus, we will reduce the size of Catalog.
+   - Bundle Mode "Pack Together by Label" will be used to differentiate between platforms (WebGL, Standalone) within the same logical group to exclude unused assets (and the related overhead) if needed
+   - We may come up with additional rules for our convenience
+
+Check [Unity guidelines](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AddressableAssetsDevelopmentCycle.html#organizing-addressable-assets) for additional information.
+
+#### Asset Bundle CRC	
+At the moment we don't have any sensitive data that requires integrity check. Moreover, for bigger files it will cause noticeable overhead on loading.
+So for the groups we will disable it.
+
+### Resources handling
+We are going to exclude Scenes and Resources from Addressables Catalog:
+   - We don't plan to migrate "Resources"' API to the corresponding one of "Addressables"
+   - We plan to migrate assets currently added to "Resources" to "Addressables" with a proper distribution over time
+
+![image](https://user-images.githubusercontent.com/118179774/214880545-4ecec0c7-6c50-4a57-97dc-946305719ae7.png)
+
+### Addressables Caching
+"Addressables" uses right the same "Caching" system that "Asset Bundles" does. It works out of the box for all the supported platforms. It is possible to disable caching per group, but in our case we will not be doing it.
+
+### Asset Reference
+We are going to utilize ["AssetReference"](https://docs.unity3d.com/Packages/com.unity.addressables@1.21/manual/AssetReferences.html) and "AssetLabelReference" instead of "SerializeField" to properly reference Assets that should be a part of an Addressable without creating a strong reference to them.
```

Comment:
this is a bit not clear for me. Do you mean that we will go with `AssetReference` via `AssetLabelReference` and not via Inspector injection (which will be forbidden, and probably even forced by the validation tests)?

Reply:
It's not a replacement for "SerializeField", it serves a different purpose: provide a reference to a separate "Addressable" that is external to the context it is referenced from.

Reply:
I added 2 use cases